src/mouse.js

```diff
@@ -58,50 +59,73 @@ var mouseEvents = {
     e.pointerId = this.POINTER_ID;
     e.isPrimary = true;
     e.pointerType = this.POINTER_TYPE;
-    if (!HAS_BUTTONS) {
-      e.buttons = WHICH_TO_BUTTONS[e.which] || 0;
-    }
     return e;
   },
+  prepareButtonsForMove: function(e, inEvent) {
+    var p = pointermap.get(this.POINTER_ID);
+    e.buttons = p ? p.buttons : 0;
+    inEvent.buttons = e.buttons;
+  },
   mousedown: function(inEvent) {
     if (!this.isEventSimulatedFromTouch(inEvent)) {
-      var p = pointermap.has(this.POINTER_ID);
+      var p = pointermap.get(this.POINTER_ID);
 
       // TODO(dfreedman) workaround for some elements not sending mouseup
       // http://crbug/149091
-      if (p) {
-        this.cancel(inEvent);
-      }
```

Comment:
With this condition being removed, should the comment be removed too, or does it still apply to the rest of the method?